code/modules/asset_cache/assets/plumbing.dm

```diff
@@ -0,0 +1,8 @@
+/datum/asset/spritesheet/plumbing
+	name = "pservice"
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names